### PR TITLE
Add blogs CRUD

### PR DESCRIPTION
**Pull request by James-nyagatare**
#### What does this PR do?
-Add crud endpoints.
#### Description of Task to be completed?
-Add create, read, update and delete routes.
-Add multer for the photo to be uploaded.
-Add cloudinary to upload the image in the cloud.
#### How should this be manually tested?
-Postman
#### Questions:
N/A

### CODE REVIEW

**Comment by Mucyo-Aime-Christian** [comment]
With my observations, this PR matches with my expectations. except if there is anything more from @harerakalex to request

File: server/src/controllers/blogController.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      );
+      if (!blogUpdated) return Response.error(res, 404, "Blog not found");
+      return Response.success(
+        res,
+        200,
+        "Sucessfully Updated the blog",
+        blogUpdated
+      );
+    } catch (error) {
+      console.log(error);
```
Comment (harerakalex) [inline server/src/controllers/blogController.js:57]: Well done @James-nyagatare, however, you should remove this console. That can give hackers information about what is happening in the system.

File: server/src/config/cloudinary.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const uploadToCloud = async (file) => {
+  try {
+    const image = await cloudinary.uploader.upload(file.path, {
+      folder: "MyBrand",
+      use_filename: true,
+    });
+    return image;
+  } catch (error) {
+    console.log(error);
```
Comment (harerakalex) [inline server/src/config/cloudinary.js]: You should return the error here instead of consoling the error.

**Review by harerakalex — CHANGES_REQUESTED** [review]
Well done @James-nyagatare, though I have got somethings that you can impprove.

**Review by harerakalex — APPROVED** [review]

**Pull request merged**